.travis.yml

```diff
@@ -1,15 +1,26 @@
 language: go
 
 sudo: false
+dist: trusty
 
 go:
-- 1.2.2
-- 1.3.3
-- 1.4
-- 1.5.4
-- 1.6.2
+- 1.x
+- 1.2.x
+- 1.3.x
+- 1.4.x
+- 1.5.x
+- 1.6.x
+- 1.7.x
 - master
 
-matrix:
-  allow_failures:
-  - go: master
+before_install:
+- find "${GOPATH%%:*}" -name '*.a' -delete
```

Comment:
I added this while troubleshooting issues with the `cover` tool, where a version that came with the place was causing issues on go 1.4.x: https://travis-ci.org/urfave/negroni/jobs/186114218#L133-L138

![](https://www.dropbox.com/s/0qupb0u07pjg81a/Screenshot%202016-12-29%2014.46.29.png?dl=1)

I'm thinking I should get this fixed upstream so that there is no `cover` tool preinstalled... 🤔